Dockerfile.import

```diff
@@ -1,10 +1,13 @@
-FROM ubuntu:xenial
+FROM debian:stretch
```

Comment:
Have you tested whatever switching to http://releases.ubuntu.com/17.04/ also fixes problem?

Note that https://github.com/gravitystorm/openstreetmap-carto/pull/2818/files switched from debian:stretch to ubuntu:xenial, I worry a bit about repeated PRs changing between ubuntu and debian, fixing one particular issue and breaking setup for somebody else.

I think that upgrading OS version would be preferable to repeated random switches (#2818, now this) to fix one particular issue.

Reply:
For info I tried the "standard" switch2osm guide on Ubuntu 17.04 and hit a problem with python-mapnik

    python
    >>> import mapnik
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
      File "/usr/lib/python2.7/dist-packages/mapnik/__init__.py", line 74, in <module>
        from ._mapnik import *
    ImportError: /usr/lib/python2.7/dist-packages/mapnik/_mapnik.x86_64-linux-gnu.so: undefined symbol: _ZTIN6mapnik15freetype_engineE

so there may be a separate issue with 17.04.

Reply:
No, I haven't tested this on 17.04. As for #2818 I'm not sure about the reason - it doesn't seem to fix anything and just switches back to an older system.

Reply:
Maybe it either forced purging caches (so PR was useless) or there was a regression somewhere so updating software caused bug to appear?

Reply:
"No, I haven't tested this on 17.04" - can you do that? Hopefully it will not run into @SomeoneElseOSM issue and ubuntu.debian dance will be avoided.

Reply:
@matkoniecz Would you please clarify, do you mean trying to switch from `debian:stretch` to `ubuntu:zesty` in the kosmtik Dockerfile?

Reply:
I think it is preferable to switch from `ubuntu:xenial` to `ubuntu:zesty` rather than from `ubuntu:xenial` to `debian:stretch` (smaller change, upgrading OS is less likely to cause regressions that changing OS).

Reply:
@matkoniecz Sure I can, while I don't see much sense in this - it's an isolated environment, who really cares about what's inside? IMHO there could be two main reasons to start changing base image:
- it's when something doesn't work (#2911)
- saving space - hence switching to the same base image in all used images

Reply:
@OnkelTem It is to avoid regressions. Switching debian:stretch to ubuntu:xenial (#2818) only to switch it back here is not useful. Especially as it is not clear what is underlying bug - both here and in #2818.